code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm

```diff
@@ -183,3 +191,65 @@
 		patrol_path = get_path_to(src, target_turf, /turf/proc/Distance_cardinal, 0, 200)
 		return
 	return ..()
+
+//On-kill visual effect
+/obj/structure/jbird_noose
+	name = "feathery noose"
+	desc = "A structure found in the black forest."
+	icon = 'ModularTegustation/Teguicons/48x64.dmi'
+	icon_state = "noose"
+	pixel_x = -8
+	base_pixel_x = -8
+	max_integrity = 60
+	buckle_lying = 0
+	density = FALSE
+	anchored = TRUE
+	can_buckle = TRUE
+
+/obj/structure/jbird_noose/attack_hand(mob/user)
+	if(has_buckled_mobs())
+		for(var/mob/living/L in buckled_mobs)
+			user_unbuckle_mob(L, user)
+	else
+		..()
+
+/obj/structure/jbird_noose/user_buckle_mob(mob/living/M, mob/user, check_loc = TRUE)
+	if(M.buckled)
+		return
+	var/response = alert(user,"Will you really hang [M]?","This looks painful","Yes","No")
+	if(response == "Yes" && do_after(user, 10, M))
+		return ..(M, user, check_loc = FALSE) //it just works
+	else
+		to_chat(user, "You decide not to hang [M].")
```

Comment:
The else statement is obsolete as you return in if() statement above. Not a code error, but rather stylistic choice we opt in for.

Reply:
Done!